google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineFacet.java

```diff
@@ -68,35 +59,4 @@ public static AppEngineFacet getAppEngineFacetByModule(@Nullable Module module)
   public AppEngineSdk getSdk() {
     return AppEngineSdkManager.getInstance().findSdk(getConfiguration().getSdkHomePath());
   }
-
-  //todo[nik] copied from JamCommonUtil
-  @Nullable
-  private static <T> T getRootElement(final PsiFile file, final Class<T> domClass,
-      final Module module) {
-    if (!(file instanceof XmlFile)) {
-      return null;
-    }
-    final DomManager domManager = DomManager.getDomManager(file.getProject());
-    final DomFileElement<DomElement> element = domManager
-        .getFileElement((XmlFile) file, DomElement.class);
-    if (element == null) {
-      return null;
-    }
-    final DomElement root = element.getRootElement();
-    if (!ReflectionUtil.isAssignable(domClass, root.getClass())) {
-      return null;
-    }
-    return (T) root;
-  }
-
-
-  public boolean shouldRunEnhancerFor(@NotNull VirtualFile file) {
```

Comment:
`shouldRunEnhancerFor` was not used in the old plugin either (I will double check that it was called outside by the framework - its not overriding anything so I don't think it is).

`getRootElement` is downstream from the old tools menu action which is now replaced by our new menu item (and we handle this DOM stuff already).